model/host/host.go

```diff
@@ -2359,3 +2359,15 @@ func FindStaticNeedsNewSSHKeys(settings *evergreen.Settings) ([]Host, error) {
 		SSHKeyNamesKey: bson.M{"$not": bson.M{"$all": names}},
 	}))
 }
+
+func (h *Host) IsSubjectToHostCreationThrottle() bool {
+	if h.UserHost {
+		return false
+	}
+
+	if h.SpawnOptions.SpawnedByTask {
+		return false
+	}
+
+	return true
```

Comment:
Docker check needed?

Reply:
More specifically, check if the host has a parent.

Reply:
maybe both?